webapp/src/components/QuestionCard.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+  TextField,
+  FormControlLabel,
+  Radio,
+  Card,
+  CardContent,
+  Typography,
+  RadioGroup,
+  FormControl,
+  MenuItem,
+} from '@mui/material';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import { Question } from '../assets/data';
+
+interface QuestionCardProps {
+  question: Question;
+  handleNewAnswer: (
+    question: Question,
+    chosenAnswerId?: number,
+    responseText?: string
+  ) => void;
+  currentStatus: string;
+}
+
+const QuestionCard = ({
+  question,
+  handleNewAnswer,
+  currentStatus,
+}: QuestionCardProps) => {
+  const [radioValue, setRadioValue] = React.useState('');
+  const handleChangeRadio = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setRadioValue(event.target.value);
+    handleNewAnswer(question, parseInt(event.target.value), undefined);
+  };
+
+  const [selectValue, setSelectValue] = React.useState('');
```

Comment:
Do we need a separate state variable for this, or can we reuse `radioValue` and `setRadioValue`?